### DISCUSSION TITLE
Anyway to plot custom indicator with custom colour ?

### DISCUSSION THREAD

**Discussion by VietnameseRick** [kernc/backtesting.py#748] [Q&A]
I am currently building a custom indicator of myself. I want to ask if the colour when bt.plot() depend on the backtesting.py or pandas_ta ?

**Comment by kernc** [comment]
The indicator color can be set by the `color=` parameter passed into [`Strategy.I()` indicator wrapper](https://kernc.github.io/backtesting.py/doc/backtesting/backtesting.html#backtesting.backtesting.Strategy.I).

By default, colors are assigned sequentially from a pool:
https://github.com/kernc/backtesting.py/blob/8b36a6b8cf30e2f8a53593b5049fbe5a71bcc833/backtesting/_plotting.py#L84-L85
https://docs.bokeh.org/en/latest/docs/reference/palettes.html#d3-palettes

**Reply by VietnameseRick** [reply]
Thank you, I am still writing my indicator. Will response when it ran successfully. 👍

**Comment by x7upLime** [comment]
Is it also possible to set the size of the scattered dots?
If not, is there any plan to implement that logic? I'd like to take a shot at it :)
